content/master/concepts/patch-and-transform.md

```diff
@@ -1654,6 +1655,25 @@ patches:
           trim: `-north-1'
 ```
 
+#### Join
+
+The {{<hover label="typeJoin" line="8">}}type: Join{{</hover>}} joins all
+values in the input array into a string using the given separator.
```

Comment:
```suggestion
values in the input array into a string using the given separator.  
```
I'd suggest adding a line break to put the line "This transform only works..." on a new line.

Reply:
Added.